addons/skin.estuary/language/resource.language.en_gb/strings.po

```diff
@@ -743,3 +743,8 @@ msgstr ""
 msgctxt "#31163"
 msgid "Show Fanart background"
 msgstr ""
+
+#: /xml/SkinSettings.xml
```

Comment:
Please also add a `#.` line explaining the purpose of this string. In the past we often "forgot" that, but translators will greatly benefit from this information.